timed/subscription/tests/test_subscription_project.py

```diff
@@ -25,7 +26,7 @@ def test_subscription_project_list(auth_client):
     # not billable reports should not be included in spent hours
     ReportFactory.create(not_billable=True, task=task,
                          duration=timedelta(hours=4))
-    # project of same customer but without a billing type with packages
+    # project of same customer but without customer_visible=True with packages
```

Comment:
sorry I did not see this before... but this is an odd comment :).

Maybe better `project of same customer but without customer_visible set`